transformer/src/testTable.ts

```diff
@@ -0,0 +1,195 @@
+import {
+  CallExpression,
+  Expression,
+  ArrayLiteralExpression,
+} from 'assemblyscript/dist/assemblyscript.js';
+import {SimpleParser} from 'visitor-as';
+import {RangeTransform} from 'visitor-as/dist/transformRange.js';
+
+/**
+ * Generates if expression.
+ *
+ * Known criteria are :
+ * - is, isNot,
+ * - isTrue, isFalse,
+ *
+ * If the criterion is not in the list,
+ * the raw string value is taken.
+ *
+ * @param {string} criterion
+ * @return {string}
+ */
+export function generateIfExpression(criterion: string) {
```

Comment:
why export this function? and we may want to put it in the class below

Reply:
true!